src/main/java/com/conveyal/gtfs/graphql/GraphQLGtfsSchema.java

```diff
@@ -421,6 +422,7 @@
             .field(MapFetcher.field("entity_id"))
             .field(MapFetcher.field("entity_sequence", GraphQLInt))
             .field(MapFetcher.field("bad_value"))
+            .field(ErrorPriorityFetcher.field("error_type_priority"))
```

Comment:
I think the intent with the `errorCountType` definition below is to contain high-level information about the error type, including the count, english message and priority fields. I think this fetcher should be removed because its redundant information. Open to hearing a counter argument though.

Reply:
I originally thought each error needed this info, but after implementing https://github.com/ibi-group/datatools-ui/pull/527 I realized I didn't really need it. So yeah, I'll remove it.